src/app/routes/getInitialData/index.js

```diff
@@ -10,15 +10,39 @@ const logger = nodeLogger(__filename);
 const STATUS_CODE_OK = 200;
 const STATUS_CODE_BAD_GATEWAY = 502;
 const STATUS_CODE_NOT_FOUND = 404;
+const RENDERER_ENV = 'renderer_env';
 const upstreamStatusCodesToPropagate = [STATUS_CODE_OK, STATUS_CODE_NOT_FOUND];
+const validRendererEnvironments = ['int', 'test', 'live'];
 
 const ampRegex = /(.amp)$/;
 
 const baseUrl = onClient()
   ? getBaseUrl(window.location.origin)
   : process.env.SIMORGH_BASE_URL;
 
-const getUrl = pathname => `${baseUrl}${pathname.replace(ampRegex, '')}.json`;
+export const getUrl = pathname => {
+  const index = pathname.indexOf(RENDERER_ENV) + 1;
```

Comment:
I see what you mean about the regex!

If you did want a regex approach to this instead of the string stuff, it would be something like:

```
const regex = /renderer_env=(.*)&?/
const [ match, environment ] = pathname.match(regex) || [];
```